dashboards/google-home-analytics/metadata.yaml

```diff
@@ -0,0 +1,39 @@
+sample_dashboards:
+  -
+    category: Google Home
+    id: google_home_analytics
```

Comment:
These ids and file names need to be aligned around either hyphens or underscores, but they should match. Precedent in prior dashboards leans toward hyphens.

Reply:
Updated!